W-17989215-DWextension-29-dm (#463)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-extension-plugin.adoc

> = Installing and Using the DataWeave 2.0 (BETA) Extension

Author: ```suggestion
= Installing and Using the DataWeave 2.0 Extension
```

> Install and use DataWeave 2.0 (BETA) extension in Visual Studio Code (DataWeave extension) to develop, preview, test, and deploy custom mappings and modules into libraries in a DataWeave project. The extension also enables you to preview transformations similar to those you perform in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground].

Author: ```suggestion
Install and use DataWeave 2.0 extension in Visual Studio Code (DataWeave extension) to develop, preview, test, and deploy custom mappings and modules into libraries in a DataWeave project. The extension also enables you to preview transformations similar to those you perform in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground].
```

== Before You Begin

Before installing the DataWeave extension to Visual Studio Code, you must:

* Install AdoptOpenJDK
* Install Apache Maven (3.6.3 minimum)
* Install Microsoft Visual Studio Code for your specific operating system

[NOTE]
If you have any other prior extensions for DataWeave, whether experimental or from a third party, uninstall them.

== Install the DataWeave Extension

To install the extension:

. Open Visual Studio Code.
. In the *EXTENSIONS* tab, type `dataweave` to search the extension in the marketplace.

> . Select *DataWeave 2.0* from the search results.

Reviewer: ```suggestion
. Select *Anypoint Code Builder - DataWeave Extension* from the search results.
```

> . Select *DataWeave 2.0 (BETA)* from the search results.

Author: ```suggestion
. Select *DataWeave 2.0* from the search results.
```
